Dockerfile

```diff
@@ -55,6 +55,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone \
 	imagemagick \
 	poppler-utils \
 	ghostscript \
+	vim \
```

Comment:
Why would we need that on a PROD container?

Reply:
It was asked by mr. Spinar...but I don't think we need it anymore.

Reply:
removed it